regenerate_api.py

```diff
@@ -161,15 +171,17 @@ def generate_api_and_copy(output_type, output_copyer, extra_commands: List[str]
             time.sleep(2)
 
     if not succeeded:
-        return
+        raise RuntimeError(
+            f'openapi generation failed after trying {n_attempts} time(s)'
+        )
 
     output_copyer(tmpdir)
     shutil.rmtree(tmpdir)
 
 
 def generate_schema_file():
     """
-    Generate schema file and place int he metamist/graphql/ directory
+    Generate schema file and place int the metamist/graphql/ directory
```

Comment:
```suggestion
    Generate schema file and place in the metamist/graphql/ directory
```

Might as well fix the whole typo :smile:

Reply:
Good catch!